vite.config.ts

```diff
@@ -9,6 +9,12 @@ export default defineConfig({
 		wasm(), 
     topLevelAwait(), 
 	],
+	worker: {
+		plugins: [
+			wasm(),
+			topLevelAwait(), 
+		],
+	},
```

Comment:
```suggestion
	worker: {
		plugins: () => [
			wasm(),
			topLevelAwait(), 
		],
	},
```

https://vite.dev/config/worker-options#worker-plugins